externalpolicy/text_language/text_language.go

```diff
@@ -20,9 +25,56 @@ func NewTextConstraintLanguagePlugin() plugin_registry.ConstraintLanguagePlugin
 
 func (p *TextConstraintLanguagePlugin) Validate(dconstraints interface{}) (bool, error) {
 
-	// Validate that the input is a ConstraintExpression type (string)
+	var err error
+	var constraints externalpolicy.ConstraintExpression
+	var nextExpression, nextLogicalOperator, remainder, constraint string
+	var validated bool
+
+	// Validate that the input is a ConstraintExpression type (string[])
+	if !isConstraintExpression(dconstraints) {
+		return false, errors.New(fmt.Sprintf("The Constrain input: %v is not Contraint Express type", dconstraints))
+	}
 
 	// Validate that the expression is syntactically correct and parse-able
+	constraints = dconstraints.(externalpolicy.ConstraintExpression)
+
+	for _, constraint = range constraints {
+		// 1 constrain inside constrain list
+		fmt.Println("constraint: ", constraint)
```

Comment:
No Printlns or Printfs...if you really want to log this then you need to use the glog.V(5).Infof(.........) API

Reply:
Thanks! This is most for debugging, will remove this line